torch/quantization/quantize.py

```diff
@@ -220,6 +226,16 @@ def quantize_qat(model, run_fn, run_args, qconfig_dict=None):
     convert(model)
     return model
 
+def quantize_dynamic(model, run_fn=None, run_args=None, qconfig_dict=None):
```

Comment:
Lets ensure that all three quantize functions take DEFAULT_MODULE_MAPPING as an argument, with a default value set. For QAT, we will need one more argument DEFAULT_QAT_MODULE_MAPPING. (cc @jerryzh168)

Reply:
yeah, we will expose this in top level quantize function.

Reply:
Exposed the `DEFAULT_MODULE_MAPPING` as an argument in this PR.